nnbuilder-api/src/main/proto/tasksqueue.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+
+package org.hse.nnbuilder.services;
+
+enum TaskName {
+  LearnNN = 0;
+  ApplyToData = 1;
+}
+
+enum TaskStatus {
+  HaveNotStarted = 0;
+  Processing = 1;
+  Done = 2;
+}
+
+service TasksQueueService {
+  rpc createtask (TaskCreationRequest) returns (TaskCreationResponse);
+  rpc getinformation(GettingInformationRequest) returns (GettingInformationResponse);
```

Comment:
Make it more specific? E.g. getTask / getTaskInfo

Reply:
why I need to separate it?